src/krate.rs

```diff
@@ -545,6 +547,16 @@ impl Crate {
         }
     }
 
+    fn remove_blacklisted_documentation_urls(url: Option<String>) -> Option<String> {
+        let blacklisted_url = "rust-ci.org";
```

Comment:
This is a great start!! Thank you!! Two improvements I'd like to see before we merge this in, however:

- I think a `contains` is a bit overzealous. While I'm definitely making up a silly case, this implementation would block someone who had their docs published at http://rust-ci.organists.com, and that would be confusing and unexpected both for crate users and for me trying to figure out what was going on. I think we should use [the url crate's `parse` function and `host_str` method](https://docs.rs/url/1.5.1/url/) to parse the documentation URL (if parsing fails, return `None`), extract the host as a string slice, and then compare against the blacklist for exact match.
- It'd be super nice if instead of one hardcoded URL, we had a hardcoded list here (that would contain one item) so that adding another URL to this blacklist would be adding that to the list, not having to change the code. The blacklist could be a constant at the top of the file so that we're not allocating the list every time we encode a crate, something like:

```
const DOCUMENTATION_BLACKLIST: [&'static str; 1] = [
    "rust-ci.org",
];
```

Then in the code, since with the first change we'll have just the host string, we actually *could* use `contains`, since then we'd be asking if the list contains the host exactly :)

Do these changes make sense? Please let me know if you have any questions!